underlay/js/bigipreport.js

```diff
@@ -3125,6 +3319,79 @@ function getDataGroup(datagroup, loadbalancer) {
     }
     return matchingdatagroup;
 }
+/** ********************************************************************************************************************
+    Returns a matching policy object from the policy json data
+**********************************************************************************************************************/
+function getPolicy(policy, loadbalancer) {
+    const policies = siteData.policy;
+    let matchingpolicy;
+    //Find the matching policy from the JSON object
+    for (const p in policies) {
+        if (policies[p].name === policy && policies[p].loadbalancer === loadbalancer) {
+            matchingpolicy = policies[p];
+        }
+    }
+    return matchingpolicy;
+}
+/** ********************************************************************************************************************
+    Shows the policy details light box
+**********************************************************************************************************************/
+function showPolicyDetails(policy, loadbalancer) {
+    //Get the policy object from the json file
+    const matchingpolicy = getPolicy(policy, loadbalancer);
+	let html;
+    //If an policy was found, prepare the data to show it
+    if (matchingpolicy) {
+        //Populate the header
+        html = '<div class="policydetailsheader">';
+        html += '<span>Policy: ' + matchingpolicy.name + '</span><br>';
```

Comment:
Please use interpolation rather than concatenation:

```
html += `<span>Policy: ${matchingpolicy.name}</span><br>`;
```
instead of:
```
html += '<span>Policy: ' + matchingpolicy.name + '</span><br>';`
```

In fact you could replace line 3346 - 3352 with one string template (back ticks). Does that make sense?

Same comment goes for the code below (prefer string interpolation to concatenation).

Reply:
changed